dataland-frontend/src/components/forms/parts/kpiSelection/DataPointFormWithToggle.vue

```diff
@@ -208,6 +212,7 @@ export default defineComponent({
   props: {
     name: {
       type: String,
+      default: "",
```

Comment:
Why? IMO there is no appropriate default.

Reply:
Because we use the name variable in the template and an error appears that name may be undefined, so if we display the component without specifying the name, there will be an empty string

Reply:
Then make it required instead. that makes more sense to me.

Reply:
changed